kafka/src/main/scala/hydra/kafka/transport/KafkaTransport.scala

```diff
@@ -68,11 +72,19 @@ class KafkaTransport(producerSettings: Map[String, ProducerSettings[Any, Any]])
           "transport" -> persistenceId
         )
       )
+
+      HydraMetrics.decrementGauge(
+        KafkaIngestor.ReconciliationGaugeName + s"_$topic",
+        KafkaIngestor.ReconciliationMetricName,
+        tags = Seq("ingestType" -> "kafka", "topic" -> topic)
```

Comment:
think you mentioned the ack level isn't available here? idk if the decrement behavior works correctly if the tags aren't congruent